src/Stryker.Core/Stryker.Core.UnitTest/Mutants/MutantOrchestratorTests.cs

```diff
@@ -6,24 +6,24 @@
 using System.Collections.ObjectModel;
 using System.IO;
 using System.Linq;
+using Stryker.Core.Compiling;
 using Xunit;
 
 namespace Stryker.Core.UnitTest.Mutants
 {
     public class MutantOrchestratorTests
     {
-        private Collection<IMutator> _mutators { get; set; }
-        private string _currentDirectory { get; set; }
-        private MutantOrchestrator _target { get; set; }
+        private string CurrentDirectory { get; set; }
```

Comment:
this is by accident: code cleanup done by R#. I kept as is, because this is the standard capitalisation for properties in c# (https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions).
![image](https://user-images.githubusercontent.com/3131866/59101514-7e787a00-8929-11e9-8698-415139ff2cfa.png)

I have seen the use of underscore for field, and I kinda like it, but never for properties (that's why R# complains about properties, but not for fields).
Can you give me links to well known repo with this pattern?

Reply:
Good point, I think we should change all these private properties to private fields. That would be more in line with the guidelines! Also R# would like that better. Should we create a seperate PR for changing to fields?

Reply:
as you want. I would simply propose to fix them on an opportunity basis: i.e. clean them up when you need to work on the class. But it should be a team decision

Reply:
I agree, clean the up when you find them.